src/pages/news/2021-04-26-OSOR-interview.en.md

```diff
@@ -0,0 +1,15 @@
+---
+path: "/en/news/2021-04-26-OSOR-interview"
+date: "2021-04-26"
+title: "OpenZaak featured in the European Open Source Observatory (OSOR)"
+lang: en
+template: default
+---
+
+# OpenZaak featured in the European Open Source Observatory (OSOR)
+
+The European portal [OSOR](https://joinup.ec.europa.eu/collection/open-source-observatory-osor), Open Source Observatory, published on April 7th an interview with Lazo Bozarov, Senior Supply Manager Digital Services at Utrecht Municipality about OpenZaak.
+
+In this text, Bozarov comments on the advantages that working in open source provides once the new [Dutch API (Application Programmable Interface) standard for case-oriented](https://www.vngrealisatie.nl/nieuws/api-standaarden-zaakgericht-werken-vanaf-1-april) has been implemented.
```

Comment:
Is a word missing here? Or does "case management" fit better than "case-oriented"?

Reply:
I've changed it!